src/Slackbot.Net.Extensions.FplBot/Helpers/GameweekEventsFormatter.cs

```diff
@@ -19,22 +21,24 @@ public static List<string> FormatNewFixtureEvents(List<FixtureEvents> newFixture
             newFixtureEvents.ForEach(newFixtureEvent =>
             {
                 var scoreHeading = $"{GetScore(context.Teams, newFixtureEvent)}\n";
-                var eventMessages = newFixtureEvent.StatMap.SelectMany(stat =>
+                var eventMessages = newFixtureEvent.StatMap
+                    .Where(stat => TeamSubscribesForThisEvent(context, stat.Key))
```

Comment:
Siden sub-filteret er inne i messageformattern og ikke utenfor [som i de andre eventene](https://github.com/fplbot/fplbot/pull/107/files#diff-a73c36bbcd737188963c740c339d74a1e965da6df972f21ceda67fc455162bc1R37)e, så vil denne returnere en tom liste. Skaper det noe humbug, eller er det håndtert?

Reply:
Håndteres her: https://github.com/fplbot/fplbot/blob/d1f6c1323d7605166dc04a22cf3fd08da371d74d/src/Slackbot.Net.Extensions.FplBot/Helpers/GameweekEventsFormatter.cs#L45

Tom liste returneres, og denne for-løkka skippes: https://github.com/fplbot/fplbot/blob/d1f6c1323d7605166dc04a22cf3fd08da371d74d/src/Slackbot.Net.Extensions.FplBot/Helpers/SlackWorkSpacePublisher.cs#L38